Controller/TwigElementsController.php

```diff
@@ -43,6 +43,16 @@ public function sideMenuAction(AssetExtractorService $assetExtractorService, Cli
         );
     }
 
+    public function jobsAction(string $username, JobService $jobService)
```

Comment:
The use of Action in the function name is deprecated since Symfony 4. Might be better to comply for new functions? And even transform in a separate pull request the other functions of this file?

Reply:
Agree but the twig controller function append an action (see the base.html.twig)